service/src/main/java/uk/gov/hmcts/reform/fpl/model/InternationalElement.java

```diff
@@ -8,14 +8,7 @@
 @Builder
 @AllArgsConstructor
 public class InternationalElement {
-    private final String issues;
-    private final String proceedings;
-    private final String issuesReason;
-    private final String possibleCarer;
-    private final String proceedingsReason;
-    private final String significantEvents;
-    private final String possibleCarerReason;
-    private final String significantEventsReason;
-    private final String internationalAuthorityInvolvement;
-    private final String internationalAuthorityInvolvementDetails;
+    private final String whichCountriesInvolved;
```

Comment:
Will removing this cause any deserialisation issues on cases with the old fields added? (Either existing cases or open cases where the flow has been partially completed?

Reply:
I think so, I'll add them back